renderer/nyaovim-app.ts

```diff
@@ -222,12 +222,20 @@ Polymer({
         argv: {
             type: Array,
             value: function() {
+                let electron_argc =  1; // the first argument of standalone distribution is the application path
+                if (remote.process.argv.length > 1
+                    && 'electron' === basename(remote.process.argv[0]).toLowerCase()) {
```

Comment:
Hmm... I can't accept a code which only works on specific platform because NyaoVim should work cross platform.  We need to check this on other platforms.